.github/workflows/rollback.yml

```
@@ -0,0 +1,76 @@
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+
+      - name: Set up SSH
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.DEPLOY_SSH_KEY }}" > ~/.ssh/deploy_key
+          chmod 600 ~/.ssh/deploy_key
+          ssh-keyscan -H ${{ secrets.DEPLOY_HOST }} >> ~/.ssh/known_hosts
+
+      - name: Verify images exist
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:92: Double quote to prevent globbing and word splitting

(shellcheck)